### PR TITLE
add new geometries from upstream

### LINKED ISSUE #262: Add additional geometries from new version of libhkl
The new versions of libhkl (v5.0.0.3001 & .3357) have some new geometries.  Add them.

### PR DESCRIPTION
**Pull request by prjemian**
- close #262

### CODE REVIEW

**Comment by prjemian** [comment]
This is ready for review.

File: docs/source/geometry_tables.rst
Comment (mrakitin) [inline docs/source/geometry_tables.rst:1]: What's the source for the tables in this document? Are the tables compiled from the data from https://people.debian.org/~picca/hkl/hkl.html#orgdf95f6f?

**Review by mrakitin — APPROVED** [review]
Looks good to me!

Out of curiosity, how are the tables generated?

File: docs/source/geometry_tables.rst
Comment (prjemian) [reply docs/source/geometry_tables.rst:1]: The tables are compiled by hand by reviewing the web documentation and the source code.

**Pull request merged**